src/util/Spotify.js

```diff
@@ -48,53 +48,66 @@ export const Spotify = {
         Tracklist.js, cannot read property 'map' of undefined, and pointing on the
         traks of the TrackList.. I couldn't understand why.*/
     async search (term) {
-      /* let userAccessToken = this.getUserAccessToken();   I commented this out becaues it is already defined in the global of this file*/
+      
+      const topic = 'Searching Spotify\'s API.'
+
+      const mainTerm = encodeURI(term); // this converts the term passed as a parameter to a URI format
+
+      let response;
+      let JSONResponse;
+      let error;
+      let mappedTracks;
 
       if (!userAccessToken){
         console.log('No Access Token.');
-        return;
+        return ;
       };
-
-      const mainTerm = encodeURI(term); // this converts the term passed as a parameter to a URI format
-      
       
+      //The code below tries to contact Spotifies API

```

Comment:
I know you write comment as a way to learn, but just know that comments, have a purpose, describe something that code fails to describe, like business logic, some algorithm for example.

Reply:
got it!
Should I delete them than?

Reply:
generally speaking yes,
unless you need them to learn

Reply:
Sababa!